### ISSUE TITLE
Events are not added to the last displayed day when week starts on Sunday

### ISSUE THREAD

**Issue by gisellemnr** [sakhnevych/CalendarView#12]
Here's a minimal example (based on the code in the README):

```
from calendar_view.calendar import Calendar
from calendar_view.core.event import EventStyles

config = data.CalendarConfig(
    lang='en',
    title='Test Schedule',
    dates='Su - Th',
    hours='8 - 20',
    mode=None,
    show_date=False,
    show_year=False,
    legend=False,
)

calendar = Calendar.build(config)
calendar.add_event(day_of_week=3, start='08:00', end='17:00', style=EventStyles.GRAY)
calendar.save("simple_view.png")
```

Does not show the event. Issues the warning:
```
WARNING:root:Skipping event, it ends before the visible range: Event[title: None, notes: None, style: EventStyle[event_border: (110, 110, 110, 240), event_fill: (200, 200, 200, 190)], day_of_week: 3, start_date: None, end_date: None, start_time: 08:00:00, end_time: 17:00:00,cascade_group: 0, cascade_total: 1, cascade_index: 1]
```

However, adding Friday or removing Monday from the week works:

```
config = data.CalendarConfig(
    lang='en',
    title='Test Schedule',
    dates='Su - Fr',
    hours='8 - 20',
    mode=None,
    show_date=False,
    show_year=False,
    legend=False,
)

calendar = Calendar.build(config)
calendar.add_event(day_of_week=3, start='08:00', end='17:00', style=EventStyles.GRAY)
calendar.save("simple_view.png")
```

or 

```
config = data.CalendarConfig(
    lang='en',
    title='Test Schedule',
    dates='Mo - Th',
    hours='8 - 20',
    mode=None,
    show_date=False,
    show_year=False,
    legend=False,
)

calendar = Calendar.build(config)
calendar.add_event(day_of_week=3, start='08:00', end='17:00', style=EventStyles.GRAY)
calendar.save("simple_view.png")
```

**Comment by sakhnevych** [comment]
Thank you for reporting an issue. Fixed in release `v2.2.0`.

**Comment by gisellemnr** [comment]
Thanks for the fix!!
